cmd/mattermost/commands/extract_content.go

```diff
@@ -69,14 +77,16 @@ func extractContentCmdF(command *cobra.Command, args []string) error {
 			break
 		}
 		for _, fileInfo := range fileInfos {
-			fmt.Println("extracting file", fileInfo.Name, fileInfo.Path)
-			err = a.ExtractContentFromFileInfo(fileInfo)
-			if err != nil {
-				mlog.Error("Failed to extract file content", mlog.Err(err), mlog.String("fileInfoId", fileInfo.Id))
+			if !ignoredFiles[fileInfo.Extension] {
+				fmt.Println("extracting file", fileInfo.Name, fileInfo.Path)
+				err = a.ExtractContentFromFileInfo(fileInfo)
+				if err != nil {
+					mlog.Error("Failed to extract file content", mlog.Err(err), mlog.String("fileInfoId", fileInfo.Id))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.